lib/http-context.js

```diff
@@ -310,10 +314,15 @@ HttpContext.prototype.done = function () {
   if (this.req.query._format) {
     accepts = this.req.query._format.toLowerCase();
   }
-  var dataExists = typeof data !== 'undefined';
 
+  var acceptableMIMEType = accepts;
+  if (acceptableMIMEType) {
+    acceptableMIMEType = acceptableMIMEType.replace(/;\s*charset=[^;]+;*/i, '');
```

Comment:
Maybe something in between, the `Accept` header can have multiple content types specified, comma separated. Each one can have specializations. Does the code that we are modifying handle multiple content types in the `Accept` header already?